README.md

```diff
@@ -163,261 +158,274 @@ service<http:Service> StudentData bind studentServiceListener {
-  #           If data is not deleted , it returns the JSON containing a status and error message.
+# Function to delete student data from database.
+# `deleteData()` is a function to delete a student's data from student records database.
+#
+# + studentId - This is the id of the student to be deleted.
+# + return -This function returns a JSON object. If data is deleted it returns JSON containing a status.
+#           If data is not deleted , it returns the JSON containing a status and error message.
 
-public function deleteData(int stuId) returns (json) {
+public function deleteData(int studentId) returns (json) {
     json status = {};
     string sqlString = "DELETE FROM student WHERE id = ?";
 
     // Delete existing data by invoking update action.
-    var returnValue = databaseEP->update(sqlString, stuId);
+    var returnValue = studentDB->update(sqlString, studentId);
     io:println(returnValue);
-    match returnValue {
-        int updateRowCount => {
-            if (updateRowCount != 1){
-                status = { "Status": "Data Not Found" };
-            }
-            else {
-                status = { "Status": "Data Deleted Successfully" };
-            }
-        }
-        error err => {
-            status = { "Status": "Data Not Deleted", "Error": err.message };
-            log:printError(err.message,err = err);
+
+    if (returnValue is int) {
+        if (returnValue != 1) {
+            status = { "Status": "Data Not Found" };
+        } else {
+            status = { "Status": "Data Deleted Successfully" };
         }
+
+    } else {
+        status = { "Status": "Data Not Deleted" };
     }
     return status;
 }
 
 # `getId()` is a function to get the Id of the student added in latest.
 #
 # + mobNo - This is the mobile number of the student added which is passed as parameter to build up the query.
-# + return -This function returns either a table which has only one row of the student details or an error.
+# + return -This function returns a table with Student type.
 
 // Function to get the generated Id of the student recently added.
-public function getId(int mobNo) returns table|error {
+public function getId(int mobNo) returns table<Student> {
     //Select data from database by invoking select action.
-    var returnValue = databaseEP->select("Select * FROM student WHERE mobNo = " + mobNo, Student, loadToMemory = true);
-    table<Student> dataTable;
-    match returnValue {
-        table tableReturned => dataTable = tableReturned;
-        error e => io:println("Select data from student table failed: " + e.message);
+
+    string sqlString = "SELECT * FROM student WHERE mobNo = ?";
+    // Retrieve student data by invoking select remote function defined in ballerina sql client
+    var ret = studentDB->select(sqlString, Student, mobNo);
+
+    table<Student> dataTable = table{};
+    if (ret is error) {
```

Comment:
Shall we handle this error? At least log the error?

Reply:
Yes, I will update it.